build.gradle

```diff
@@ -721,18 +861,33 @@ project(':connect:file') {
     enabled = false
   }
 
-  tasks.create(name: "copyDependantLibs", type: Copy) {
-    from (configurations.testRuntime) {
-      include('slf4j-log4j12*')
+  tasks.create(name: "copyDependantLibsJava", type: Copy) {
```

Comment:
Since these tasks are the same (or should be) for all modules can they be de-duplicated and moved to the subprojects section of the build?